src/command_modules/azure-cli-acr/README.rst

```diff
@@ -9,54 +9,55 @@ Commands to manage Azure container registries
         az acr: Commands to manage Azure container registries.
 
     Subgroups:
-        credential: Manage admin user credential for Azure container registries.
+        credential: Manage administrator login credentials for Azure container registries.
         repository: Manage repositories for Azure container registries.
 
     Commands:
-        check-name: Check whether the container registry name is available.
-        create    : Create a container registry.
-        delete    : Delete a container registry.
-        list      : List container registries.
-        show      : Get a container registry.
-        update    : Update a container registry.
+        check-name: Checks whether the container registry name is available for use.
+        create    : Creates or updates a container registry with the specified parameters.
```

Comment:
I agreed that it might be confusing to the user. This description was meant to match what the actual behavior is. If the user "creates" an existing registry with different properties, it is indeed an update, while in update command, all parameters are optional. VM has similar behaviors in their create command.